### DISCUSSION TITLE
Hidding Object using GameObjectFilter

### DISCUSSION THREAD

**Discussion by Rbo99** [yasirkula/UnityRuntimeInspector#36] [Q&A] [answered]
Hi there, 
i would like to hide some Game Object in the Hierarchy. How could i do ?
If it's by using the GameObjectFilter, how should i use it ?
Thanks by advanced

**Comment by yasirkula** [comment, marked as answer]
Here's an example: https://forum.unity.com/threads/runtime-inspector-and-hierarchy-open-source.501220/page-4#post-5393046

**Reply by Rbo99** [reply]
Thanks for your quick answer. And thanks for this asset :)
